### PR TITLE
Adding Donation Tracker and Thank you randomizer

### LINKED ISSUE #18: Donation Tracker (+ Thank you notes)
Attending to pending feature request:

- Have donors manually input the amount they contributed
- Randomly generate a thank you message afterwards!

### PR DESCRIPTION
**Pull request by vunderkind**
This is a Draft PR. Backend migration must be complete before this feature can be greenlit.

### CODE REVIEW

File: src/components/UserCard.js [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+        const randomthanks = thanks[Math.floor(Math.random() * thanks.length)]
+        return randomthanks
+        
+    }
+
+    
+    async function postDonation(){
+    const response = await axios.put(
+        `https://good-faith.herokuapp.com/api/people/29`,
+        { donationAmount: "1000.00" },
```
Comment (oreHGA) [inline src/components/UserCard.js]: @vunderkind I think you omitted making these request params dynamic

Comment (oreHGA) [reply src/components/UserCard.js]: - `donationAmount` should get what's in the input field
- request endpoints should reference the dynamic "recipients" id

**Review by oreHGA — CHANGES_REQUESTED** [review]
Nice one!

Couple comments:
- could you add `.DS_Store` to the gitignore and not track it here
- left some comments on the `postDonation` method in `UserCard` component

**Comment by vunderkind** [comment]
Thanks Ore. Wilco.

File: src/components/UserCard/UserCard.css [hunk 2]
```diff
@@ -20,4 +20,55 @@ details {
...
+    width: 100vw;
+    border-radius: 15px 15px 15px 0;
+}
+
+.Donated p {
+    margin: 0;
+    line-height: 1;
+}
+
+.Button {
```
Comment (obasekietinosa) [inline src/components/UserCard/UserCard.css:49]: For this, might consider using the button component? Perhaps adding a new type tertiary in Button.css
Would improve maintainability.

File: src/components/UserCard/UserCard.js [hunk 3]
```diff
@@ -29,14 +69,39 @@ function UserCard(props) {
...
+                            <input 
+                            type="text"
+                            placeholder="2000"
+                            value={donated.value} 
+                            onChange={handleChange} 
+                            className="Input">
+                            </input>:null}
+                            {donated.value && donated.value===Number?null:null}
+                            </label><br/>
+                            {checked.status? <button onClick={postDonation} className="Button">Submit</button>: null}
```
Comment (obasekietinosa) [inline src/components/UserCard/UserCard.js]: Button utility component might help here.

File: src/components/UserCard/UserCard.css (hunk 2 above)
Comment (vunderkind) [reply src/components/UserCard/UserCard.css:49]: Thanks Etin. I'll consider fixing it in a separate PR. Tried using the button component, but it wasn't working, visually.

**Pull request merged**